src/manifolds/GeneralUnitaryMatrices.jl

```diff
@@ -473,17 +473,48 @@ function get_vector_orthonormal!(
 end
 
 @doc raw"""
-    injectivity_radius(G::Orthogonal)
-    injectivity_radius(M::Rotations)
+    injectivity_radius(G::GeneraliUnitaryMatrices)
+
+Return the injectivity radius for general unitary matrix manifolds, which is
+
+````math
+    \operatorname{inj}_{\mathrm{U}(n)}(p) = π.
+````
+"""
+injectivity_radius(::GeneralUnitaryMatrices) = π
+
+@doc raw"""
+    injectivity_radius(G::GeneraliUnitaryMatrices{n,𝔽,DeterminantOneMatrices})
 
-Return the injectivity radius on the [`Rotations`](@ref) and the [`OrthogonalMatrices`](@ref) `M`,
-which is globally
+Return the injectivity radius for general unitary matrix manifolds, where the determinant is $+1$,
+which is
 
 ````math
-    \operatorname{inj}_{\mathrm{O}(n)}(p) = \operatorname{inj}_{\mathrm{SO}(n)}(p) = π\sqrt{2}.
+    \operatorname{inj}_{\mathrm{SU}(n)}(p) = π.
```

Comment:
```suggestion
    \operatorname{inj}_{\mathrm{SU}(n)}(p) = π \sqrt{2}.
```

Reply:
Oh, I was careful, but still I mixed up one in the docs.